components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/AbstractHTTPFunction.java

```diff
@@ -174,6 +174,7 @@ protected void executeHttpMethod(HttpUriRequest clientRequest, Map<String, Objec
             if (!isRetry.shouldRetry()) {
                 return result;
             }
+            outcome = result.getRight().getLeft();
```

Comment:
can there be any null pointer exceptions?

Reply:
no